### DISCUSSION TITLE
Verfügbarkeit

### DISCUSSION THREAD

**Discussion by 2Abendsegler** [2Abendsegler/GClh#1890] [General]
Bin ab morgen für eine Woche nicht verfügbar, nur vielleicht mal abends.

**Comment by capoaira** [comment]
Moin,
tut mir leid, dass ich im Moment nicht wirklich aktiv bin. Seit dem mein Studium online stattfindet, bin ich so viel vorm PC, dass ich kaum Motivation habe meine Freizeit auch am Schreibtisch zu verbringen. Aber ich bin weiterhin gerne dabei und würde PRs testen und kleine Issues übernehmen. Irgendwann darf ich dann hoffentlich auch wieder **in** der Uni studieren ;) und habe mehr Freizeit.

Apostroph Uni: Mein erstes Semester ist nun fast vorbei, und ich habe in dieser Zeit gemerkt, wie viel ich bei diesem Projekt gelernt habe. Ich danke dir für die viele Unterstützung und Hilfe, durch die ich eine Menge gelernt habe. Es macht Spaß, mit dir zu arbeiten 😊

**Comment by 2Abendsegler** [comment]
Moin,
danke für den lieben Kommentar. 

Es geht völlig in Ordnung, dass du seit Beginn des Studiums hier etwas kürzer getreten bist. Und dass sich das durchs Homeoffice noch verstärkt hat, kann ich gut nachvollziehen. Alles gut!

Ich hatte dich mit Absicht etwas außen vor gelassen, um keinen Druck aufzubauen. Davon gibt es an anderen Stellen ja genug, brauchen wir hier nicht auch noch. Ich werde dich bei Gelegenheit wieder beim Testen einplanen.

Im Moment habe ich selbst nicht so viel Zeit und auch nicht so viel Motivation hier etwas zu machen. Die Bugs mit den Log Type Countern und die Bugs bei den PQs müssen deshalb wohl noch etwas auf ihre Korrektur warten. Das ist ok. Eigentlich auch gut, dass ich die Bugs bei den PQs selbst verursacht habe, das bringt mich gerade etwas vom Perfektionismus weg. 🤭

Ich arbeite auch gerne mit dir zusammen. Und du bist in vielerlei Hinsicht eine Bereicherung in diesem Projekt, danke dir. 😊
  
LG
